src/coprocessor/dag/handler.rs

```diff
@@ -111,9 +111,16 @@ impl DAGRequestHandler {
             eval_cfg.set_sql_mode(SqlMode::from_bits_truncate(req.get_sql_mode()));
         }
 
-        let is_batch = enable_batch_if_possible
-            && !is_streaming
-            && super::builder::DAGBuilder::can_build_batch(req.get_executors());
+        let mut is_batch = false;
+        if enable_batch_if_possible && !is_streaming {
+            let build_batch_result =
+                super::builder::DAGBuilder::check_build_batch(req.get_executors());
+            if let Err(e) = build_batch_result {
+                debug!("Coprocessor request cannot be batched"; "reason" => %e);
```

Comment:
I think the `debug` level may be not enough here.

Reply:
Do you like it to be `info`?

Pros: We can see this log in production without any modifications
Cons: Maybe too many log output before we fully implemented batch execution (maybe before GA)

Reply:
@AndreMouche What do you think?

Reply:
I think we should use `debug` here.